pages/index.js

```diff
@@ -81,11 +125,16 @@ export default function HomePage({
   handleToolTip,
   emotionEntries,
   theme,
+  toggleExampleData,
+  useExampleData,
 }) {
   const [isFormSubmitted, setIsFormSubmitted] = useState(false);
   const [id, setId] = useState();
   const [tension, setTension] = useState(0);
-  const [chartIsShown, setChartIsShown] = useState(true);
+  const [chartIsShown, setChartIsShown] = useState(false);
+  const [showInfoBox, setShowInfoBox] = useState(false);
+
+  const newest_ID = emotionEntries[emotionEntries.length - 1]._id;
```

Comment:
Since we always want to use camelCase for variable naming, you could think about a more appropriate name, e.g. newestEntryId / latestEntryId ...